python/ml4ir/applications/classification/tests/data/configs/model_config.yaml

```diff
@@ -16,4 +16,7 @@ layers:
     rate: 0.0
   - type: dense
     name: final_dense
-    activation: null
\ No newline at end of file
+    activation: null
+calibration:  
```

Comment:
@marjanhs I've removed it from the default config and added it here. You can test it using ->
```
python3 ml4ir/applications/classification/pipeline.py \
--data_dir ml4ir/applications/classification/tests/data/tfrecord \
--feature_config ml4ir/applications/classification/tests/data/configs/feature_config.yaml \
--model_config ml4ir/applications/classification/tests/data/configs/model_config.yaml \
--batch_size 32 \
--run_id test \
--data_format tfrecord \
--execution_mode train_inference_evaluate
```